[2022-06-27 13:31:07,650] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-06-27 13:31:07,675] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-02-07T00:00:00+00:00 [queued]>
[2022-06-27 13:31:07,679] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:07,683] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:31:07,685] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:07,714] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_A> on 2022-02-07 00:00:00+00:00
[2022-06-27 13:31:07,721] {standard_task_runner.py:52} INFO - Started process 57030 to run task
[2022-06-27 13:31:07,747] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_A', 'scheduled__2022-02-07T00:00:00+00:00', '--job-id', '195', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpeoy3873r', '--error-file', '/tmp/tmpu9ldny0h']
[2022-06-27 13:31:07,753] {standard_task_runner.py:80} INFO - Job 195: Subtask model_A
[2022-06-27 13:31:07,858] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_A scheduled__2022-02-07T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:31:07,972] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-06-27 13:31:07,974] {python.py:173} INFO - Done. Returned value was: 7
[2022-06-27 13:31:08,081] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_A, execution_date=20220207T000000, start_date=20220627T133107, end_date=20220627T133108
[2022-06-27 13:31:08,194] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:31:08,281] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
